SPECS/bc/bc.spec

```diff
@@ -1,30 +1,37 @@
-Summary:	precision numeric processing language
-Name:		bc
-Version:	1.07.1
-Release:        3%{?dist}
-License:	GPLv2+
-URL:		https://ftp.gnu.org/gnu/bc/
-Group:		System Environment/base
+Summary:        precision numeric processing language
+Name:           bc
+Version:        1.07.1
+Release:        4%{?dist}
+License:        GPLv2+
+URL:            https://ftp.gnu.org/gnu/bc/
```

Comment:
Nit: While we're here, can we change the URL to something like the [GNU bc homepage](https://www.gnu.org/software/bc/) or the [GNU bc manual](https://www.gnu.org/software/bc/manual/bc.html)?

Reply:
good idea, will change to the homepage